src/OWML.Logging/ConsoleUtils.cs

```diff
@@ -6,7 +6,7 @@ namespace OWML.Logging
 {
     public static class ConsoleUtils
     {
-        private static Dictionary<MessageType, ConsoleColor> _messageTypeColors = new Dictionary<MessageType, ConsoleColor> {
+        private static readonly Dictionary<MessageType, ConsoleColor> MessageTypeColors = new Dictionary<MessageType, ConsoleColor> {
```

Comment:
thought we had the style that private variables were "_likeThis" or "likeThis" and public variables were "LikeThis" ? doesn't really matter

Reply:
the standard for private static fields/properties is CamelCase. I'm not a huge fan of it either, but... :P